src/components/Auth/InputField/Input/Input.tsx

```diff
@@ -0,0 +1,33 @@
+import { ChangeEvent, FocusEventHandler } from "react";
+import styles from "./Input.module.scss";
+
+interface Input {
+  className: string;
+  type: string;
+  name: string;
+  value: any;
+  onValueChange: (e: ChangeEvent<HTMLInputElement>) => void;
+  onInputFocus: FocusEventHandler;
+  onInputBlur: FocusEventHandler;
+  placeholder?: string;
+}
+
+export default function Input({
+  className,
+  onValueChange,
+  placeholder,
+  onInputBlur,
+  onInputFocus,
+  ...rest
+}: Input) {
```

Comment:
타입명은 이렇게 표현되는게 좋을 것 같네요!
```suggestion
}: InputProps) {
```